js/util/mapbox.js

```diff
@@ -2,78 +2,87 @@
 
 var config = require('./config');
 var browser = require('./browser');
+var url = require('url');
+var querystring = require('querystring');
 
-function normalizeURL(url, pathPrefix, accessToken) {
+function normalizeURL(parsedUrl, pathPrefix, accessToken) {
```

Comment:
Would this module be simpler if we parsed the url within `normalizeURL` instead of within its callers?